core/urls.py

```diff
@@ -16,8 +16,16 @@
 """
 
 from django.contrib import admin
-from django.urls import path
+from django.conf import settings
+from django.conf.urls.static import static
+from django.urls import path, include
+# Assuming bid is the app name
+from bid.views import home, upload_property, get_properties, get_csrf
 
 urlpatterns = [
+    path('', home, name='home'),
     path("admin/", admin.site.urls),
-]
+    path('upload_property/', upload_property, name='upload_property'),
```

Comment:
following standard API design, it's best practice to group actions related to the same resource under a single endpoint. for example if we're building GET and POST operations managing the `property` entities, using `/properties/` is a better option.

```python
# urls.py

urlpatterns = [
    # ... other paths ...
    path('properties/', PropertyView.as_view(), name='properties'),
]
```

```python
# views.py

from django.views import View


class PropertyView(View):
    def get(self, request):
        # <logic for retrieving a property>
        return xxx
    def post(self, request):
        # <logic for uploading a property>
        return xxx
```

ref: django docs for [class-based-views](https://docs.djangoproject.com/en/5.0/topics/class-based-views/intro/#using-class-based-views)
disclaimer: just suggestions from a higher-level, code itself might be wrong for implementations. and it's totally fine if you want to leave them unchanged because they DO work 👍🏻

Reply:
But I have upload and get, and I want to make a distinction.